test/one/profiler/test/Output.java

```diff
@@ -51,6 +60,20 @@ public double ratio(String regex) {
         return (double) matched / total;
     }
 
+    public Output convertFlameToCollapsed() throws IOException {
+        FlameGraph fg = new FlameGraph(new Arguments("-o", "collapsed"));
+        try (InputStreamReader in = new InputStreamReader(
+                new ByteArrayInputStream(toString().getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8)) {
```

Comment:
This double UTF-8 conversion String->bytes then bytes->String bothers me a bit.
Perhaps, it's better to use `StringReader` instead.

Reply:
Addressed